LogIn/LoginViewController.swift

```diff
@@ -10,32 +10,54 @@ import Combine
 
 class LoginViewController: UIViewController {
     
+    var model: LoginViewModelType
+    
+    private let email = PassthroughSubject<String, Never>()
```

Comment:
To remove boilerplate and copy-paste, its nice to write extension to the TextField with custom publisher.

Reply:
Check out https://github.com/CombineCommunity/CombineCocoa/blob/main/Sources/CombineCocoa/Controls/UITextField%2BCombine.swift as an example reference

Reply:
good one!
Done